tests/standalone/ClickUsing.py

```diff
@@ -0,0 +1,16 @@
+import click
+
+# nuitka-skip-unless-imports: click
+
+@click.command()
+#@click.option("--count", default = 1, help = "Number of greetings.")
+#@click.option("--name", prompt = "Your name", help = "The person to greet.")
+
+def hello(count = 5, name = "Tabi"):
```

Comment:
In the method signature, there should be no spaces between the variable name, equals symbol and the value.